File: ios/stereoCamera/stereoCamera/Comm/Commands/ConnectionPause.swift
//
//  ConnectionPause.swift
//  stereoCamera
//
//  Created by hallmarklabs on 7/9/18.
//  Copyright © 2018 cody. All rights reserved.
//

import Foundation

class ConnectionPause : Command
{
    override init()
    {
        super.init()
        cmdtype = CommandTypes.CONNECTION_PAUSE
        expectsResponse = false
    }
}

File: android/app/src/main/java/com/munger/stereocamera/ip/command/commands/ConnectionPause.java
package com.munger.stereocamera.ip.command.commands;

import com.munger.stereocamera.ip.command.Command;

public class ConnectionPause extends Command
{
	public ConnectionPause()
	{
		super();
		cmdtype = Type.CONNECTION_PAUSE;
		expectsResponse = false;
	}
}
